modules/apps/headless/headless-foundation/headless-foundation-impl/rest-openapi.yaml

```diff
@@ -626,6 +626,14 @@ paths:
           name: pageSize
           schema:
             type: integer
+        - in: query
+          name: filter
+          schema:
+            type: string
+        - in: query
+          name: sort
```

Comment:
I think the yaml file should be sort by name

Reply:
There are more unsorted keys, we will sort it once, with SF.